src/OceanSimulations/OceanSimulations.jl

```diff
@@ -25,6 +25,7 @@ end
 
 Default() = Default(nothing)
 default_or_override(default::Default, value=default.value) = value
+default_or_override(user_specified_value::Number, args...) = user_specified_value
```

Comment:
This isn't quite right. Can I commit to this PR? I have an idea to make the design better.

Reply:
for sure